TestCases/parallel_regression.py

```diff
@@ -502,6 +510,15 @@ def main():
     turb_naca0012_sst_expliciteuler.timeout   = 3200
     test_list.append(turb_naca0012_sst_expliciteuler)
 
+    # NACA0012 (SST, 2003m, U-MUSCL)
+    turb_naca0012_sst_umuscl           = TestCase('turb_naca0012_sst_umuscl')
+    turb_naca0012_sst_umuscl.cfg_dir   = "rans/naca0012"
+    turb_naca0012_sst_umuscl.cfg_file  = "turb_NACA0012_sst_umuscl.cfg"
+    turb_naca0012_sst_umuscl.test_iter = 10
+    turb_naca0012_sst_umuscl.test_vals = [-5.562109, -5.896601, 0.802894, 1.087476, 0.020323, -2.567948, 0.000000]
+    turb_naca0012_sst_umuscl.timeout   = 3200
+    test_list.append(turb_naca0012_sst_umuscl)
```

Comment:
Modify some old cases instead of adding new ones, if one of the V&V cases could use this (flat plate maybe?) it would be nice too so that we track converged results.